Source/HTTP/WinHttp/winhttp_connection.cpp

```diff
@@ -1366,6 +1366,7 @@ void CALLBACK WinHttpConnection::completion_callback(
 #if HC_PLATFORM != HC_PLATFORM_GDK
 HRESULT WinHttpConnection::set_autodiscover_proxy()
 {
+#if !defined(HC_WINHTTP_WIN32_NOXASYNC)
```

Comment:
>HC_WINHTTP_WIN32_NOXASYNC [](http://example.com/codeflow?start=13&length=25)

where is this getting defined?

Reply:
by whoever wants to build the win32 dll w/o xasync